src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.controller.js

```diff
@@ -35,7 +35,11 @@ angular.module("umbraco")
     	    });
 
     	    helpService.findVideos(rq).then(function(videos){
-    	    	$scope.videos = videos;
+    	        $scope.videos = videos;
+
+    	        angular.forEach($scope.videos, function (obj) {
+    	            obj.thumbnail = obj.thumbnail.replace(/(\.[\w\d_-]+)$/i, '_thumb$1');
+    	        });
```

Comment:
we handle these images in a remote rss feed, so its a valid issue, but something we will handle outside of the core  - I think you will need to do a new PR as these one contains quite a few different things in one big pile